designsafe/static/scripts/projects/components/leave-feedback-modal/leave-feedback-modal.template.html

```diff
@@ -0,0 +1,44 @@
+<div class="modal-header" style="background: #e6e6e6;">
+    <h4 class="modal-title">
+        Leave Feedback
+        <span class="pull-right" ng-click="$ctrl.close()">
+            <i class="fa fa-times">&nbsp;</i>
+        </span>
+    </h4>
+</div>
+<div class="modal-body">
+    <div class="loading bg-info" ng-if="$ctrl._ui.loading">
+        <i class="fa fa-spinner fa-spin">&nbsp;</i> Loading...
+    </div>
+    <div class="alert alert-error" ng-if="$ctrl.error">
+        <p>There was a problem submitting your feedback.</p>
+    </div>
+    <div class="alert alert-success" ng-if="$ctrl.success">
+        <p>Your feedback was submitted!</p>
+    </div>
+    <form name="feedback">
+        <fieldset ng-disabled="$ctrl.submitting">
+            <label for="feedback-name">Full Name</label>
+            <input name="feedback-name" type="text" style="width:100%;" ng-model="$ctrl.feedbackForm.name" ng-required="true" ng-disabled="$ctrl.Django.context.authenticated" />
+            <br>
+            <br>
+            <label for="feedback-email">Email</label>
+            <input name="feedback-email" type="text" style="width:100%;" ng-model="$ctrl.feedbackForm.email" ng-required="true" ng-disabled="$ctrl.Django.context.authenticated" />
```

Comment:
Since this is going into RT, should we have form validation?  In case the reviewer needs to be contacted back.